### PR TITLE
TOMEE-2580 - Translation of the examples/simple-cdi-interceptor to Portuguese.

### PR DESCRIPTION
**Pull request by gabrielferreirapro**
Hey guys there is my translation of simple-cdi-interceptor and some fixes in English version.
Thank you ;)
jira task: https://issues.apache.org/jira/browse/TOMEE-2580

### CODE REVIEW

File: examples/simple-cdi-interceptor/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+@Retention(RUNTIME)
+public @interface Log {
+}
+....
+
+Tenha certeza que você não esqueceu da anotação `@InterceptorBinding` acima! 
+Agora que nossa anotação customizada foi criada, vamos anexa-la (ou "vincula-la") 
+a um interceptador.
+
+Aqui esta nosso interceptador de log. Um método `@AroundInvoke` e estamos quase
```
Comment (marcoantoniobferreira) [inline examples/simple-cdi-interceptor/README_pt.adoc]: "Aqui esta" the correct is "Aqui está"

File: examples/simple-cdi-interceptor/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    @AroundInvoke
+    public Object logMethodEntry(InvocationContext ctx) throws Exception {
+        System.out.println("Entering method: " + ctx.getMethod().getName());
+        //or logger.info statement 
+        return ctx.proceed();
+    }
+}
+....
+
+Agora a anotação `@Log` que criamos esta vinculada a este interceptador.
```
Comment (marcoantoniobferreira) [inline examples/simple-cdi-interceptor/README_pt.adoc]: "que criamos esta vinculada" the correct is "que criamos está vinculada"

File: examples/simple-cdi-interceptor/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        return ticketPrice - 50;
+    }
+    // assume more methods are present
+}
+....
+
+A anotação `@Log` aplicada em nível de classe diz que todos os métodos desta 
+classe devem ser interceptados pelo `LoggingInterceptor`.
+
+Antes de dizer-mos "tudo pronto" temos que fazer uma ultima coisa! Habilitar
```
Comment (marcoantoniobferreira) [inline examples/simple-cdi-interceptor/README_pt.adoc]: "dizer-mos" the correct is "dizermos".
"ultima" the correct is "última"

**Comment by gabrielferreirapro** [comment]
Thank you for you review @marcoantoniobferreira !
I fixed the mistakes.

**Pull request merged**